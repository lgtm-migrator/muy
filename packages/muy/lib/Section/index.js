import React from "react"
import Box from "@material-ui/core/Box"

const Section = ({ display, ...props }) => (
  <Box component={"section"} {...props} />
)

export default Section

Section.defaultProps = {
  display: "block",
}
