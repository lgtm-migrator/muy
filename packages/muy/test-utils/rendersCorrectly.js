import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
