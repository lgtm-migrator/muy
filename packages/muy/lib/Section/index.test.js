import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Section from "."

describe("<Section /> component", () => {
  it("should render correctly with no props", () => {
    rendersCorrectly(<Section />)
  })
})
