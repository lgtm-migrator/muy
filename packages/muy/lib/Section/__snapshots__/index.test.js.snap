// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> component should render correctly with no props 1`] = `
<section
  className="MuiBox-root MuiBox-root-2"
/>
`;
